src/components/screens/Sessions.tsx

```diff
@@ -111,9 +156,12 @@ export default function Sessions() {
 
   return (
     <Flex flexDirection="column" width="100%" height="100%" mt={8}>
-      {hasProposals ? (
+      {hasProposals && (
```

Comment:
Can you keep the ternary please
It's always better to return null when you don't want to render something instead of a value that can be a boolean or a value because it could be printed in the dom if it's a string for example